tests/decomposition/test_transforms_decompose.py

```diff
@@ -0,0 +1,291 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for the new experimental graph-based decomposition system integrated with `qml.transforms.decompose`."""
+
+# pylint: disable=no-name-in-module
+
+from functools import partial
+from unittest.mock import patch
+
+import pytest
```

Comment:
```suggestion
import numpy as np
import pytest
```

Reply:
Fixed all the codecov warnings [5a0e691](https://github.com/PennyLaneAI/pennylane/pull/6966/commits/5a0e691090acf85027c88cad0c81bf845e8eb603)